pages/Identity/Overview.md

```diff
@@ -7,22 +7,222 @@ menu: Identity
+
+## Ownership
+
+On the blockchain ownership is determined by the ability to sign messages.
+To this end, ownership of an identity is also determined by the address of the creator.
+(Options for allowing a delegate to create an identity on behalf of someone is described in the [Identity Factory](/Identity/Factory).)
+
+Creation of any new identity MUST be authorized by the owner's address.
+The official [Identity Factory](/Identity/Factory) must be used for initial creation of a new identity.
+
+Ownership is managed though [EIP-173](https://eips.ethereum.org/EIPS/eip-173) which provides methods that confirm ownership and facilitate ownership transfer.
+
+## Delegation
+
+Delegation allows adding public keys that are allowed to sign announcements
+or perform other actions in addition to the "owner" public key. 
+
+### Option 1: Without Permissions
+
+In this interface, the implementation determines permissions allowed for delegates.
+This permission would need to be managed separately.
+
+```solidity
+interface IDelegation {
+    
+    /**
+     * @dev Log for addition of a new delegate
+     * @param delegate Address delegated
+     */
+    event DSNPAddDelegate(address delegate);
+
+    /**
+     * @dev Log for removal of a delegate
+     * @param delegate Address revoked 
+     * @param endBlock Block number considered to be the end of the delegate permissions
+     */
+    event DSNPRemoveDelegate(address delegate, uint64 endBlock);
+
+    /**
+     * @dev Add Delegate (default permissions determined by implementation)
+     * @param newDelegate Address to delegate new permissions to 
+     * 
+     * MUST be called from owner or other delegate with permissions (implementation specific)
+     * MUST consider newDelegate to be valid from the beginning to time
+     * MUST emit DSNPAddDelegate
+     */
+    function delegate(address newDelegate) external;
+
+    /**
+     * @dev Add Delegate (default permissions determined by implementation)
+     * @param delegate Address to remove all permissions from
+     * @param endBlock Block number to consider the permissions terminated (MUST be > 0x0). 
+     * 
+     * MUST store endBlock for response in isAuthorizedToAnnounce
+     * MUST emit DSNPRemoveDelegate
+     */
+    function removeDelegate(address delegate, uint64 endBlock) external;
+
+    /**
+     * @dev Checks to see if address is authorized to announce messages
+     * @param addr Address that is used to test against erecover
+     * @param blockNumber Check for authorization at a particular block number
+     * @return boolean 
+     * 
+     * @dev Return MAY change as deauthorization can revoke past messages
+     */
+    function isAuthorizedToAnnounce(address addr, uint64 blockNumber) external view returns (bool);
+}
+```
+
+### Option 2: With Permissions
+
+In this interface, the implementation determines permissions allowed for delegates.
+This permission would need to be managed separately.
+
```

Comment:
Does this leave room for updating the contract to allow changing of permissions, or adding new permissions for a delegated address?

Reply:
Yes, but it would not be standard across the interface. So if you deploy your /own/ DSNP Identity compatible contract, you can do what you want.

Perhaps I need to be more clear about that...

Reply:
Attempted some better wording.